services/movie_session.py

```diff
@@ -42,3 +42,11 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list[dict]:
+    movie_session = MovieSession.objects.get(id=movie_session_id)
+    return [
+        {"row": ticket.row, "seat": ticket.seat}
+        for ticket in movie_session.ticket_set.all()
+    ]
```

Comment:
Retrieve a list of required dicts in one line using Django ORM. You [could use](https://docs.djangoproject.com/en/5.0/ref/models/querysets/#values) `.values` for it